### DISCUSSION TITLE
Update bonusblocks on Claim into MySQL

### DISCUSSION THREAD

**Discussion by soepketel-dev** [GriefPrevention/GriefPrevention#815] [Q&A] [answered]
I have enabled MySQL Database support.

When a player has 0 bonusblocks (and accruedblocks) they can't claim a piece of land. I created a website where they can convert in-game balance into bonusblocks. When I do an simple UPDATE query and set the bonusblocks to (example) 100, the plugin doesnt see that change. It still thinks the user has 0 bonusblocks.

When a user has a claim of 100 blocks, and I update the sql value of bonusblocks to 200 (when it was 50 before) it overwrites it to 150 on UnClaim. while the expected behaviour is to set the bonusblocks value to 300 (200 current value + 100 unclaim)

How can this be fixed?

**Comment by Jikoo** [comment]
GP caches values so it isn't querying the database for everything constantly. Your best bet would probably be to run a console command to adjust bonus claim blocks instead of modifying the database.

**Comment by soepketel-dev** [comment]
@Jikoo but a player must be online to run the command for additional bonusblocks?

**Comment by Jikoo** [comment, marked as answer]
Target players can be offline for `/adjustclaimblocks`. You do need a way to get the command to the console, but I imagine that's already something you have the capability to do if you're setting up some kind of web-based shop.

**Comment by Jikoo** [comment]
Closing this due to lack of response - assuming running console commands has resolved your issue.
